src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation/style.css

```diff
@@ -0,0 +1,423 @@
+.docVis {
```

Comment:
no comments here and in palce it is used that this is generated and very temporary. BIG WARNING COMMENT should be here and there. Plus info about how it will change in the future (description of the planned development plan)